aesop/aesop-gel
# fix(pid-182): Use correct input type for password component

Fixes https://aesoponline.atlassian.net/browse/PID-182?jql=project%20%3D%20PID, 
which was causing Chrome on Android to incorrectly display the password field, as well as not allowing password managers to auto-fill the field. 

To test: 
- `npm start`
- Open the `PasswordInput` component - verify that it fills in with a password manager.
